media-gfx/viewnior: Bump to version 1.7

tastytea opened the pull request:
Hello friends,
I'd like to maintain media-gfx/viewnior. It is orphaned. I believe by rewriting the ebuild to use meson and incorporating a bugfix I have proven that I can write ebuilds. The bugfix is a [commit from upstream](https://github.com/hellosiyan/Viewnior/commit/aac0a0c4ad232723e8aecbab4dff99691b90a08e) after the last release.

juippis commented:
Hey, 

yes this looks good enough to count. It builds and runs fine, too. Few minor things in the ebuild that I'll point out next, that you most likely have to fix before getting this merged.

tastytea commented on media-gfx/viewnior/viewnior-1.7.ebuild:
```diff
@@ -0,0 +1,57 @@
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-libs/glib:2
```
I'm not sure what you mean by this. `gio` is part of glib, `x11-libs/gdk-pixbuf` is a dependency of `x11-libs/gtk+:2`. Should I add `virtual/pkgconfig` to `DEPEND`?

juippis replied:
>Should I add virtual/pkgconfig to DEPEND?

Yes. I also wanted you to confirm that the other deps gets pulled, since I didn't know ;)

For what it matters, looks good to me. Builds and works fine.

tastytea replied:
I have updated the ebuild.

tastytea commented:
I have updated the ebuild.

* I do not have the consent of the upstream maintainer on being added in metadata.xml and therefore I removed it.
* I checked, gnome2-utils inherits xdg-utils. Removed it.
* Added `|| die`.

tastytea commented on media-gfx/viewnior/viewnior-1.7.ebuild:
```diff
@@ -0,0 +1,60 @@
+	virtual/pkgconfig
+"
+RDEPEND="${COMMON_DEPEND}
+	dev-util/glib-utils
```
Fixed.

tastytea commented:
Umm… Why is there suddenly an ebuild for viewnior 1.7 (with QA issues, btw) in the tree, @radhermit? Are you gonna maintain the package now?

a17r commented on media-gfx/viewnior/viewnior-1.7.ebuild:
```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-inherit xdg-utils gnome2-utils meson
```
I explained above why this was bad advice. Please add it back. ;)

tastytea commented:
I have updated the ebuild.

a17r commented:
Thanks for your work, I'm going to fix RDEPEND sorting/style after merge.
